spec/classroom_spec.rb

```diff
@@ -0,0 +1,27 @@
+require_relative '../classroom'
+
+describe Classroom do
+  before(:each) do
+    @book = Classroom.new('Classroom')
+  end
+
+  it 'creates a new rental' do
+    expect(@book).to be_an_instance_of(Classroom)
+  end
```

Comment:
- [ ] I believe the description of this test does not align with what you're testing. You're testing the classroom code and classrooms don't have any relationship with rentals. Please modify your tests to actually match the Classroom class methods 👍